src/app/business/delfin/storage-details/storage-details.component.html

```diff
@@ -445,9 +447,45 @@
                     </div>
                 </ng-template>
             </p-tabPanel>
+            <p-tabPanel header="Performance Monitoring">
+                <iframe [src]="performanceMonitorURL | safe"
+                width="100%" height="750" frameborder="0"></iframe>
+            </p-tabPanel>
         </p-tabView>
     </div>
     
     
 </div>
+<p-dialog styleClass="perf-metrics-config-dialog"  header="Configure Performance Metrics Collection" [(visible)]="showperfMetricsConfigForm" [width]="700" modal="modal">
+    <form *ngIf="showperfMetricsConfigForm" [formGroup]="perfMetricsConfigForm" [grid]="{label: 'ui-g-5', content:'ui-g-7'}" [errorMessage]="errorMessage">
+       
+        <div class="ui-g">
+            <div class="ui-g-12">
+                <form-item label="{{perfMetricsConfigFormLabel.perf_collection}}" >
+                    <p-inputSwitch formControlName="perf_collection" onLabel="Yes" offLabel="No"></p-inputSwitch>
+                </form-item>
+            </div>
+        </div>
+        <div class="ui-g">
+            <div class="ui-g-12">
+                <form-item label="{{perfMetricsConfigFormLabel.interval}}">
+                    <input formControlName="interval" value="10" type="number" size="30" pInputText> 
+                </form-item>
+            </div>
+        </div>
+        <div class="ui-g">
+            <div class="ui-g-12">
+                <form-item label="{{perfMetricsConfigFormLabel.is_historic}}" >
+                    <p-inputSwitch formControlName="is_historic" onLabel="Yes" offLabel="No"></p-inputSwitch>
+                    <span><i pTooltip="historic collection is the collection all availabe timestamps from previous polling to current polling, also supported only if the backend supports historic collection " tooltipPosition="right" class="fa fa-question-circle"></i></span>
```

Comment:
Can we make it little short, as it's tool tip?
"If backend supports historic collection, enabling this will collect data periodically"

Reply:
Fixed this.